packages/bazel/src/ng_module.bzl

```
@@ -80,28 +73,25 @@ def _enable_ivy_value(ctx):
     strategy = compile_strategy(ctx)
     if strategy == "legacy":
         return False
-    elif strategy == "global":
-        return True
     elif strategy == "aot":
         return "ngtsc"
-    elif strategy == "jit":
-        return "tsc"
     else:
         fail("unreachable")
 
 def _include_ng_files(ctx):
-    """Determines whether Angular outputs will be produced by the current compilation strategy.
+    """Determines whether Angular outputs will be produced by the ng_module rule.
 
     Args:
       ctx: skylark rule execution context
 
     Returns:
-      true iff the current compilation strategy will produce View Engine compilation outputs (such as
-      factory files), false otherwise
+      true iff the target was tagged with "ivy-only" label, false otherwise
```

I believe this comment is inverted:

```suggestion
      false iff the target was tagged with "ivy-only" label, true otherwise
```

---

good catch. thanks